tests/test_parallel_transcription.py

```
@@ -0,0 +1,239 @@
+#!/usr/bin/env python3
+"""
+Test script for parallel transcription functionality
+Tests the new 10-minute chunk limits and parallel processing
+"""
+
+import os
+import sys
+import time
```

Ruff (0.11.9)

`time` imported but unused

Remove unused import: `time`

(F401)